force-app/main/default/classes/UnitTestHelper.cls

```diff
@@ -0,0 +1,72 @@
+@isTest
+public with sharing class UnitTestHelper 
+{
+    private static final String OPP_NAME_SUFFIX = '- Test Opp';
+    private static final String OPP_DEFAULT_STAGE = 'Prospecting';
+    private static final Integer OPP_AMOUNT_INCREMENTOR = 1000;
+
+    public static String RANDOM_DOMAIN = UserInfo.getOrganizationId() + '.' + Integer.valueOf(math.rint(math.random()*1000000));
+
+    //Return 
+    public static List<Opportunity> createOpps_InMemory(Account testAcc, Integer nrOpps)
```

Comment:
I think this could have a better name as it does more than just create the requested number of opportunities in memory - it also increases the opportunity amount in a linear fashion. 

It's always a good idea for unit test helper methods to be as granular as possible and leave the logic to the test method in question. So in this case I would have a helper method that creates a single opportunity based on the parameters that are passed to it, and have the unit test method iterate through the desired number of opportunities and change the amounts (and possibly other values). That way someone can look at the unit test and understand exactly what is being set up, without having to go to the test helper and look at that code too.